### ISSUE TITLE
Set up authorization framework

### ISSUE THREAD

**Issue by Mei152** [Mei152/hours_import#9]
Install Pundit gem.

Roles will be:
- admin
- user

**Comment by mkwiatkowski** [comment]
Authorization is something that you have to think about when implementing all features. You can already see this by some issues' descriptions where you list what is available/visible to the user and what an admin can access.

I'd name this ticket "Setup authorization framework" and change the label to "task". This is probably one of the first tickets you may want to do, as others may depend on pundit being already there.

**Comment by Mei152** [comment]
I implemented your recommendations.

**Comment by Mei152** [comment]
I just burned up a lot of time trying to figure out why Pundit was giving me unable to find policy errors. Finally turned out that all I needed to do was restart the server. This inspired some questions.

I thought spring kind of handles that sort of thing for you. What's different about policy changes?

How is it that part of the app, and Pundit, seemed to have partial awareness of each other (throwing policy errors), but the awareness was not complete in that Pundit could not find the policies until the server was restarted? Put another way, what are the different parts of Pundit's architecture that do, and do not, require a server restart to become known, and properly interact, with the rest of rails?
